From 60446a17e3c97daf5f05c01ef3268678edf3bdd0 Mon Sep 17 00:00:00 2001
From: kevin shen <b00984@freescale.com>
Date: Fri, 18 Apr 2008 11:40:29 +0800
Subject: [PATCH] ENGR00073340 CR fix mx27 compile error

add a empty function for on-off key configure for mx27 and mx31

Signed-off-by: Kevin Shen <b00984@freescale.com>
---
 arch/arm/mach-mx27/mx27ads_gpio.c |    7 ++++++-
 arch/arm/mach-mx3/mx31ads_gpio.c  |    7 ++++++-
 2 files changed, 12 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx27/mx27ads_gpio.c b/arch/arm/mach-mx27/mx27ads_gpio.c
index 7936499..d1d0200 100644
--- a/arch/arm/mach-mx27/mx27ads_gpio.c
+++ b/arch/arm/mach-mx27/mx27ads_gpio.c
@@ -1,5 +1,5 @@
 /*
- *  Copyright 2004-2007 Freescale Semiconductor, Inc. All Rights Reserved.
+ *  Copyright 2004-2008 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -160,6 +160,11 @@ void gpio_uart_inactive(int port, int no_irda)
 	}
 }
 
+void gpio_on_off_button_active(void)
+{
+}
+EXPORT_SYMBOL(gpio_on_off_button_active);
+
 /*!
  * Configure the IOMUX GPR register to receive shared SDMA UART events
  *
diff --git a/arch/arm/mach-mx3/mx31ads_gpio.c b/arch/arm/mach-mx3/mx31ads_gpio.c
index 39cc5fd..ece2853 100644
--- a/arch/arm/mach-mx3/mx31ads_gpio.c
+++ b/arch/arm/mach-mx3/mx31ads_gpio.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2007 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2008 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -346,6 +346,11 @@ void gpio_keypad_inactive(void)
 
 EXPORT_SYMBOL(gpio_keypad_inactive);
 
+void gpio_on_off_button_active(void)
+{
+}
+EXPORT_SYMBOL(gpio_on_off_button_active);
+
 /*!
  * Setup GPIO for a CSPI device to be active
  *
-- 
1.5.4.4

